server/routes/homepage.js

```diff
@@ -64,6 +64,7 @@ const createHomepageRouter = ({ cmsService, offenderService }) => {
         },
         hideSignInLink: true,
         title: 'Home',
+        updates: recentlyAddedHomepageContent.data,
```

Comment:
This probably isn't needed until we know the actual query and we should deconstruct the data for each of the update panels when we have that data and pass those into the view?

Reply:
Sorry, left over from the dummy mockups 👍